include/z64audio.h

```diff
@@ -129,17 +131,25 @@ typedef struct {
     /* 0x08 */ s16 book[1]; // size 8 * order * npredictors. 8-byte aligned
 } AdpcmBook; // size >= 0x8
 
+/**
+ * Meta-Data for samples.
+ * Found inside soundfonts
+ */
 typedef struct {
-    /* 0x00 */ u32 codec : 4;
-    /* 0x00 */ u32 medium : 2;
+    /* 0x00 */ u32 codec : 4; // Type of compression used for the sample
```

Comment:
`CODEC_S16` is like the `COMPRESSION_NONE` option of the group, where raw 16-bit pcm data is loaded

Reply:
`REVERB` as well is happens to mean "uncompressed data" because once an ADPCM sample is decompressed in RAM, the uncompressed sample is stored in a `reverbBuffer` in ram where it lies in wait until it's brought back to make an "echo". That's just a rough sketch I believe though, reverb is tough to understand

Reply:
Yeah but I mean there are different kinds of "decompressed" so it's not just compression?

Reply:
Well `codec` stands for `compression-decompression` (or `coder-decoder`) so it just depends on the context of the sentence then. You can interpret these as the state of "compression" of the samples, or the "decompression" needed.

Reply:
😅 I don't mean decompression vs compression,
I mean different formats of uncompressed data along with different formats of compressed data
Which makes me say the codec isn't purely about the formats of compressed data